sycamore/transforms/partition.py

```diff
@@ -359,7 +359,7 @@ def __init__(self, model_name_or_path, threshold: float = 0.4):
         pass
 
     def partition(self, document: Document) -> Document:
-        from sycamore.transforms import SycamorePDFPartitioner
+        from sycamore.transforms.detr_partitioner import SycamorePDFPartitioner
```

Comment:
Just confirming that just importing this doesn't pull in any dependencies or anything that are platform specific? (e.g. cuda stuff).

Reply:
no.